src/components/Dialog/BaseDialog/BaseDialog.tsx

```diff
@@ -31,13 +31,15 @@ export const BaseDialog: FC<BaseDialogProps> = React.forwardRef(
             actionsClassNames,
             dialogWrapperClassNames,
             dialogClassNames,
+            positionStrategy = 'absolute',
+            style,
```

Comment:
It was coming fixed by default when we tested, changing it to absolute by default won't affect where it is already used?

Reply:
Since by default the element is portaled to the body, position absolute will also work. But in cases it doesnt people can change positionStrategy